src/main/java/org/threadly/litesockets/NoThreadSocketExecuter.java

```diff
@@ -70,6 +70,29 @@ public void setClientOperations(final Client client) {
     }
     commonSelector.wakeup();
   }
+  
+  @Override
+  public void startListening(final Server server) {
+    if(!checkServer(server)) {
+      return;
+    } else {
+      if(server.getServerType() == WireProtocol.TCP) {
```

Comment:
Seems like the only thing chosen in these conditionals is the accept/write/read.  Maybe it makes sense to set that to a variable so that the construction and execution code is not duplicated.

minor nit.